src/search/pages/NoResult/NoResult.style.ts

```diff
@@ -24,6 +26,9 @@ export const StyledNoResultDescription = styled.p`
   font-style: normal;
   font-weight: 500;
   line-height: 130%; /* 1.95rem */
+
+  display: inline;
+  white-space: nowrap;
```

Comment:
```suggestion
  display: inline-block;
```
`display` 속성을 `inline-block`으로 명시하면 줄바꿈이 더 자연스럽게 되네요😮

`에 대한 검색결과가 없습니다.` 텍스트가 일종의 박스처럼 동작해서 줄바꿈이 텍스트 전체 단위로 되는 것 같은데 `inline-block` 요소의 줄바꿈에 대한 문서를 못찾겠네요🤔

쏠이 한번 찾아보는걸로..

Reply:
<img width="803" alt="스크린샷 2024-08-28 오후 10 46 22" src="https://github.com/user-attachments/assets/fffa6874-3d27-4170-be1a-bc7b9cc39c53">

위 사진이 `inline + nowrap`이고 밑에 사진이 `inline-block`입니다
처음엔 위 사진을 의도하고 inline을 썼지만... 지금보니까 밑이 더 나은 것 같아 [적극반영](https://github.com/yourssu/Soomsil-Web/pull/290/commits/b6b4a8fe87c2bfad9d64f903dc00b602c51e1326) 했습니다!
[여기에](https://stackoverflow.com/questions/9189810/css-display-inline-vs-inline-block) 있는 사진을 보면 쭌 말처럼 텍스트가 박스처럼 동작한다고 생각을 하면 될 것 같습니다! 
block이 줄바꿈을 텍스트 전체 단위로 해주니까 nowrap 속성을 제거하고 하나만 써도 되는 것 같아요 (아니면 어떡하지)